Viewer.cpp

```diff
@@ -271,7 +271,7 @@ void Viewer::openDoc(const char *path) {
 	bool success = d->open(path);
 	doc = std::move(d);
 	if (!success) {
-		show_msgbox("nPDF", "Can't open document");
+		throw "Can't open document.";
```

Comment:
Not sure whether it's worth it to start using exceptions for this. Where is this actually caught?

Reply:
It would be caught in `main()`. I just extended the logic that was already present for the creation of the `Viewer` instance to make it more consistent.